src/server/views/layout.html

```diff
@@ -29,6 +31,16 @@
   <link rel="preload" as="font" href="{{ assetPath }}/fonts/bold-b542beb274-v2.woff2" type="font/woff2" crossorigin="anonymous">
   <link rel="preload" as="font" href="{{ assetPath }}/fonts/light-94a07e06a1-v2.woff2" type="font/woff2" crossorigin="anonymous">
   <link rel="stylesheet" href="{{ getAssetPath("stylesheets/application.scss") }}">
+
+  {% if googleAnalyticsTrackingId and cookieConsent === true %}
+    <script data-testid="ga-tag-js-main" async src="https://www.googletagmanager.com/gtag/js?id={{ googleAnalyticsTrackingId }}"></script>
+    <script data-testid="ga-tag-js-init">
```

Comment:
Because Google Tag Manager can include custom JavaScript, we can't risk putting this in `<head>`

We do add `async` here yet whilst it's non-blocking it'll still run immediately when ready

It's execution may impact lazy-loaded `async import()` scripts like the Accessible Autocomplete

Let's move it to `bodyEnd` instead and add `defer` at the very least

Reply:
Note we'll have to test this, afaik it will impact the analytics gathered. Should be fine considering the other improvements.

Reply:
Yeah it'll 100% affect the analytics as it'll run after our components do

That's what we want?